src/main/java/step3/domain/ConfirmResults.java

```diff
@@ -0,0 +1,66 @@
+package step3.domain;
+
+import step3.constants.PrizeGrade;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class ConfirmResults {
+
+	private final Map<PrizeGrade, Integer> groupedByPrizeGrade;
+
+	private static final int ZERO = 0;
+	private static final int INCREASE = 1;
```

Comment:
상수 다음 인스턴스 필드가 오는게 컨벤션입니다!
```suggestion
	private static final int ZERO = 0;
	private static final int INCREASE = 1;
	
	private final Map<PrizeGrade, Integer> groupedByPrizeGrade;
```